doc/usage/extensions/autosummary.rst

```diff
@@ -285,8 +285,12 @@ The following variables available in the templates:
 
 .. data:: attributes
 
-   List containing names of "public" attributes in the class.  Only available
-   for classes.
+   List containing names of "public" attributes in the class/module.  Only
+   available for classes and modules.
+
+    .. versionchanged:: 3.1
+
+    Attributes of modules are supported.
```

Comment:
This content should be indented inside the `versionchanged` directive. But it's minor bug. So I'll work this after merging.